### PR TITLE
Ev 56 fix search users bugs

### PR DESCRIPTION
**Pull request by ghost**
### Jira

* [56](https://chita.atlassian.net/jira/software/projects/EV/boards/1?selectedIssue=EV-56)

### Description

Fix some bugs after Demo 2

### Summary of change

increase margins and paddings
delete dead code

### Checklist

- [x] Includes a link to JIRA ticket and all related PR (API dependencies etc.)
- [x] Includes a brief description of what has been changed
- [x] Test coverage for **new** code doesn't decrease the existing test coverage
- [x] Lint errors/warnings fixed
- [x] New code is documented if necessary
- [x] Dev Testing has been successfully finished
- [x] Reviewed by 2 developer on your team
- [x] Reviewed by 1 expert

### CODE REVIEW

File: src/components/SearchUsers/SearchUsers.js [hunk 1]
```diff
@@ -25,9 +25,12 @@ const SearchUsers = ({
...
   const [page, setPage] = useState(1);
 
-  const { totalPages } = users.data.pageViewModel;
+  const {
+    items,
+    pageViewModel: { totalPages, pageNumber },
+  } = users.data;
 
-  const getUsers = pageNumber => getSearchUsersDispatch(pageNumber);
+  const getUsers = folio => getSearchUsersDispatch(folio);
```
Comment (MelenkoOleksandr) [inline src/components/SearchUsers/SearchUsers.js]: what is folio?

Comment (MelenkoOleksandr) [reply src/components/SearchUsers/SearchUsers.js]: Change param naming

**Review by PopowychSasha — DISMISSED** [review]

File: src/store/initialState.js [hunk 2]
```diff
@@ -56,10 +56,8 @@ const initialState = {
     count: null,
     editedUser: null,
     userSearchFilter: null,
-    data: {
-      items: [],
-      pageViewModel: {},
-    },
+    items: [],
+    pageViewModel: {},
```
Comment (MelenkoOleksandr) [inline src/store/initialState.js]: keep the initial state near the reducer

File: src/components/SearchUsers/SearchUsers.js [hunk 3]
```diff
@@ -25,9 +25,12 @@ const SearchUsers = ({
   const { pathname } = useLocation();
   const [page, setPage] = useState(1);
 
-  const { totalPages } = users.data.pageViewModel;
+  const {
+    items,
+    pageViewModel: { totalPages, pageNumber },
+  } = users;
```
Comment (MelenkoOleksandr) [inline src/components/SearchUsers/SearchUsers.js]: Why not get these props in mapStateToProps? 
Also create wrapper for this connect

**Pull request closed without merging**